python/variants/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from pyspark.sql import SparkSession
+from pyspark.tests import ReusedPySparkTestCase
+
+from variants import VariantsContext
+
+
+class SampleTestWithPySparkTestCase(ReusedPySparkTestCase):
+    def test_variants_context(self):
+        self.spark = SparkSession(self.sc)
```

Comment:
Rename the class to `VariantSparkAPITestCase` and the method to `test_importance_analysis_from_vcf`